core/src/kvs/lq_v2_doc.rs

```diff
@@ -344,7 +344,6 @@ mod test_check_lqs_and_send_notifications {
 		let mut stm = LiveStatement::new(Fields::all());
 		let mut session: BTreeMap<String, Value> = BTreeMap::new();
 		session.insert(OBJ_PATH_AUTH.to_string(), Value::Strand(Strand::from("auth")));
-		session.insert(OBJ_PATH_SCOPE.to_string(), Value::Strand(Strand::from("scope")));
```

Comment:
Do we need the `record` parameter here?

Reply:
Sorry, yes, I meant `access`. I think it's intuitive to expose the `access` variable in any place that we also expose the `auth` variable.

Reply:
This is no trouble at all, so I will do it :+1: